api/api/src/main/scala/uk/ac/wellcome/platform/api/services/WorksRequestBuilder.scala

```diff
@@ -176,5 +180,7 @@ object WorksRequestBuilder extends ElasticsearchRequestBuilder {
         termsQuery(
           field = "data.items.locations.locationType.id",
           values = itemLocationTypeIds)
+      case PartOfFilter(id) =>
+        termQuery(field = "state.relations.ancestors.id", value = id)
```

Comment:
Just to double check: the partOf display field is constructed from the `ancestors` field at query time, right? There's not an internal field called partOf we should do a terms query on instead?

Reply:
Yes that's right: the reason `ancestors` isn't just called `partOf` is that `ancestors` is a flattened list, whilst `partOf` is nested work data structure